setup.py

```diff
@@ -7,7 +7,9 @@
     from distutils.core import setup
 
 install_reqs = ['cached-property',
-                'python-dateutil', ]
+                'python-dateutil',
+                "six == 1.15.0"]
```

Comment:
I think we probably want a loose version pin here. Users of the module might already have a different version of six in their requirements.

Reply:
I have changed to `six >= 1.13.0` as that is the minimum version that will work.